app/src/main/java/org/openobservatory/ooniprobe/test/test/WebConnectivity.java

```diff
@@ -35,11 +35,14 @@ public WebConnectivity() {
 	}
 
 	@Override public int getRuntime(PreferenceManager pm) {
-		if (getMax_runtime() != null)
+		if (getMax_runtime() != null  &&
+				!getMax_runtime().equals(PreferenceManager.MAX_RUNTIME_DISABLED))
```

Comment:
Can we check for the max runtime being >= 0 here?

Reply:
Same as iOS PR. You should know the logic of setting runtime by the user more in deep

Reply:
I actually don't and find it very convoluted

Reply:
The user can set values starting from 11 in this field. So I find more readable checking for `MAX_RUNTIME_DISABLED` rather than <= 0